weather_sp/splitter_pipeline/file_splitters.py

```diff
@@ -68,20 +61,19 @@ def _copy_dataset_to_storage(self, src_file: t.IO, target: str):
         with FileSystems().create(target) as dest_file:
             shutil.copyfileobj(src_file, dest_file)
 
-    def _get_output_file_path(self, key: SplitKey) -> str:
-        split_keys = key._asdict()
-        if self.output_info.output_dir and key.levelType:
-            split_keys['levelType'] = f'{key.levelType}_'
-        return self.output_info.file_name_template.format(**split_keys)
+    def _get_split_dims(self) -> t.List[str]:
+        all_format = list(filter(None, [field[1] for field in string.Formatter().parse(
+            self.output_info.unformatted_output_path())]))
+        return [key for key in all_format if not key.isdigit()]
```

Comment:
Is this still used? How does it compare to the split_dims on the dataclass?

Reply:
Oops! Forgot to clean that up! Thanks for spotting this!